rhizome/host/lib/vm_setup.rb

```diff
@@ -47,7 +47,6 @@ def vp
   def prep(unix_user, public_key, nics, gua, ip4, local_ip4, boot_image, max_vcpus, cpu_topology, mem_gib, ndp_needed, storage_params, storage_secrets, swap_size_bytes, pci_devices)
     setup_networking(false, gua, ip4, local_ip4, nics, ndp_needed, multiqueue: max_vcpus > 1)
     cloudinit(unix_user, public_key, nics, swap_size_bytes)
-    download_boot_image(boot_image)
```

Comment:
We can also deprecate `boot_image` parameter. It's not used in this file anymore. The `boot_image` info passed via `storage_params`

Reply:
I removed it.